bin/setup_exercism_local_aws

```diff
@@ -97,4 +97,43 @@ end
       }
     ]
   )
+
+  Exercism.config.dynamodb_client.update_table(
```

Comment:
Unless I'm missing something `dynamodb_client` doesn't actually exist as a method in this repo. This code only works because the place that you're running it from another app that does have it defined.

```suggestion
  ExercismConfig::SetupDynamoDBClient.().update_table(
```

Reply:
See comment above

Reply:
OK, that's fine for this one then 🙂